src/main/scala/uk/gov/hmrc/initprototype/GithubManager.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.initprototype
+
+import play.api.libs.json._
+import scalaj.http.{Http, HttpResponse}
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future, blocking}
+import scala.util.{Success, Try}
+
+class GithubManager(config: GithubConfiguration)(implicit ec: ExecutionContext) {
```

Comment:
@ellamdav @JoPintoPaul what do you think about renaming the managers to connectors, so GithubConnector instead? Not essential, just did a double take and wasn't sure what manager meant, where connector feels like it's go the role of "sends messages to external services"